src/Middleware/CSP/README.md

```diff
@@ -0,0 +1,34 @@
+# CSP
```

Comment:
all of these files should be moved into a top-level CSP folder (otherwise we'll just be adding a src/ directory to their repo!) and this README.md should go in that top folder